### DISCUSSION TITLE
blog/setup-active-directory-one-way-trust-with-selective-authentication/

### DISCUSSION THREAD

**Discussion by giscus** [hanpq/getps.dev#15] [Comments]
# blog/setup-active-directory-one-way-trust-with-selective-authentication/

Setting up a one-way trust with selective authentication and without DNS conditional forward

https://getps.dev/blog/setup-active-directory-one-way-trust-with-selective-authentication/

**Comment by leviu-ofek** [comment]
Great article. Thank you!

Did you mean port 445 for SMB and not 455?

**Reply by hanpq** [reply]
Hi!

Thanks!

Yes that's a typo. I'll update the post, thanks! :)

Kind regards
Hannes
